src/Uno.UI/UI/Xaml/Input/PassThroughPseudoGestureRecognizer.iOS.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Linq;
+using Windows.UI.Xaml;
+using Foundation;
+using UIKit;
+
+namespace Uno.UI.Xaml.Input
+{
+	/// <summary>
+	/// A pseudo (discrete) gesture recognizer used to forward pointer events received on a native only view
+	/// to its managed logical <see cref="FrameworkElement.Parent"/>.
+	/// </summary>
+	/// <remarks>
+	/// This is used for native views which has a gesture recognizer which request to not forward events to its parents (a.k.a. super views).
+	/// It's not the case for all native views, but only few specific views that have a custom pointers handling, like UIScrollView.
+	/// </remarks>
+	/// <remarks>
+	/// This will **NOT** apply any offset on the touches coordinates.
+	/// It means that the native view to which is associated this pseudo recognizer is expected to be fully stretched in its logical parent's bounds.
+	/// </remarks>
+	/// <remarks>
+	/// This is a discrete gesture recognizer, that means it won't prevent other gesture recognizers registered on the same view to kick-in.
+	/// This recognizer will actually always stay in the <see cref="UIGestureRecognizerState.Possible"/> state
+	/// (except in case of <see cref="UIGestureRecognizer.TouchesCancelled"/> where it will go in <see cref="UIGestureRecognizerState.Cancelled"/> as required by Apple's documentation).
+	/// </remarks>
+	public class PassThroughPseudoGestureRecognizer : UIGestureRecognizer
```

Comment:
could this be internal?

Reply:
Yeah it could, but it also makes sens that some application would need it (and could not build it by themselves since it's using some internal API), that's why I put it in the `Uno.UI` namespace.

But anyway it was not working as expected (the bubbling was not happening in the right order), so it has been removed!